translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+    }
+  },
+  "config guilds remove-rank": {
+    "name": "rang-törlése",
+    "description": "Töröl egy Hypixel guild rangját.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang van."
+      },
+      {
+        "name": "rang",
+        "description": "A törölnivaló rang."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "removedRank": [
+        "A **{rank}** rang a **{guild}** nevű guildban törölve lett."
+      ]
+    }
+  },
+  "config guilds add-requirement": {
+    "name": "követelmény-hozzáadása",
+    "description": "Hozzáad egy követelményt egy Hypixel guild-hez.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild, amihez a követelményt adod."
+      },
+      {
+        "name": "követelmény",
+        "description": "A hozzáadandó követelmény.",
+        "choices": [
+          "Hypixel szint",
+          "Képesség átlag",
+          "Slayer tapasztalat",
+          "Minion helyek",
+          "Catacombs szint",
+          "Fairy Souls",
+          "Súlyozás",
+          "API beállítások bekapcsolva"
+        ]
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény értéke. Írj be akármilyen számot a 'API beállítások bekapcsolva' követelményhez."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "requirementNotFound": [
+        "A **{requirement}** követelmény nem létezik a **{guild}** nevű guild-ban."
+      ],
+      "addedRequirement": [
+        "A **{requirement}** követelmény hozzá lett adva a **{guild}** nevű guild-hoz!"
+      ]
+    }
+  },
+  "config guilds remove-requirement": {
+    "name": "követelmény-eltávolítás",
+    "description": "Eltávolít egy követelményt egy Hypixel guild-ról.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild, ahonnan eltávolítod a követelményt."
+      },
+      {
+        "name": "követelmény",
+        "description": "Az eltávolítandó követelmény.",
+        "choices": [
+          "Hypixel szint",
+          "Képesség átlag",
+          "Slayer tapasztalat",
+          "Minion helyek",
+          "Catacombs szint",
+          "Fairy Souls",
+          "Súlyozás",
+          "API beállítások bekapcsolva"
+        ]
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`